Tests/scripts/collect_tests_and_content_packs.py

```diff
@@ -1151,9 +1156,22 @@ def get_from_version_and_to_version_bounderies(all_modified_files_paths: set, id
         (string, string). The boundaries of the lowest from version (defaults to 0.0.0)
          and highest to version (defaults to 99.99.99)
     """
+    modified_metadata_list = modified_metadata_list if modified_metadata_list else set([])
     max_to_version = LooseVersion('0.0.0')
     min_from_version = LooseVersion('99.99.99')
     max_from_version = LooseVersion('0.0.0')
+
+    for metadata in modified_metadata_list:
+        pack_metadata_path = os.path.join(tools.pack_name_to_path(metadata), PACKS_PACK_META_FILE_NAME)
+        pack_metadata = get_pack_metadata(pack_metadata_path)
+        from_version = pack_metadata.get('serverMinVersion')
+        to_version = pack_metadata.get('serverMaxVersion')
+        if from_version:
+            min_from_version = min(min_from_version, LooseVersion(from_version))
+            max_from_version = max(max_from_version, LooseVersion(from_version))
+        if to_version:
+            max_to_version = max(max_to_version, LooseVersion(to_version))
+
```

Comment:
Lets make this a different method so we can add unittests for it

Reply:
Especially the case where we have the beta word in the value

Reply:
1) no reason to split to another function, I will add unit tests to it in the new PR.
2) `beta` version is handled by `LooseVersion` but I will add a test for it too.